tensorflow/tools/pip_package/BUILD

```diff
@@ -244,7 +244,7 @@ filegroup(
         "@nlohmann_json_lib//:LICENSE.MIT",
         "@tbb//:LICENSE",
     ]) + if_rocm([
-        "@rocprim_archive//:LICENSE.TXT",
+        "@rocprim_archive//:LICENSE.txt",
```

Comment:
Please split this license change into a separate PR.
O do not think the two changes are related, and this change can go in today.
`if_cuda` issue will take some work to resolve.

Reply:
done. PR #28119 filed.  will rebase this PR once that one is merged. thanks